test/dotnet-new3.UnitTests/DotnetNewInstantiate.cs

```diff
@@ -591,5 +594,37 @@ public void DoesNotReportErrorOnDefaultUpdateCheckOfLocalPackageDuringInstantiat
                 .And.NotHaveStdErr()
                 .And.HaveStdOutContaining("File actions would have been taken:");
         }
+
+        [Theory]
+        [InlineData("no options", new[] { "TestAssets.TemplateWithBooleanParameters" })]
+        [InlineData("options without value", new[] { "TestAssets.TemplateWithBooleanParameters", "--A", "--B", "--C", "--D" })]
+        [InlineData("option equals to false", new[] { "TestAssets.TemplateWithBooleanParameters", "--A", "false", "--B", "false", "--C", "false", "--D", "false" })]
+        [InlineData("option equals to true", new[] { "TestAssets.TemplateWithBooleanParameters", "--A", "true", "--B", "true", "--C", "true", "--D", "true" })]
```

Comment:
Should we add testcases for unspecified value and missing default value?

Reply:
This should be covered by parameter A: https://github.com/dotnet/templating/pull/5728/files#r1053118152 or do you mean something else here?